fragdenstaat_de/templates/froide_legalaction/legaldecision_detail.html

```diff
@@ -91,11 +93,13 @@ <h3 class="h5 mt-0 mb-1">
           </li>
 
           {% if object.foi_document %}
-            <iframe src="{% url 'filingcabinet:document-detail_embed' pk=object.foi_document.id slug=object.foi_document.slug %}" frameborder="0" style="width: 100%; height: 90vh; border: 0;"></iframe>
+            {% get_pdf_viewer object.foi_document %}
+
+            {% include "_frontend.html" with entry_point="filingcabinet.js" %}
```

Comment:
maybe there's a better way to let the template tag handle the static files import?

Reply:
Sorry, not really. The `_frontend.html` include way is currently the best way I found to include an entrypoint and their dependencies.